subprojects/signing/src/integTest/groovy/org/gradle/plugins/signing/SigningPublicationsIntegrationSpec.groovy

```diff
@@ -0,0 +1,244 @@
+        given:
+        buildFile << """
+            apply plugin: 'ivy-publish'
+            apply plugin: 'maven-publish'
+            ${keyInfo.addAsPropertiesScript()}
+
+            publishing {
+                publications {
+                    ivy(IvyPublication) {
+                        from components.java
+                    }
+                    maven(MavenPublication) {
+                        from components.java
+                    }
+                }
+            }
+
+            signing {
+                ${signingConfiguration()}
+                sign publishing.publications.ivy, publishing.publications.maven
+            }
+        """
+
+        when:
+        enableGradleMetadata()
+
+        and:
+        run "signIvyPublication", "signMavenPublication"
+
+        then:
+        ":signIvyPublication" in nonSkippedTasks
+        ":signMavenPublication" in nonSkippedTasks
+
+        and:
+        file("build", "libs", "sign-1.0.jar.asc").text
+        file("build", "publications", "maven", "module.json.asc").text
+        file("build", "publications", "ivy", "module.json.asc").text
+    }
+
+    @IgnoreIf({GradleContextualExecuter.parallel})
+    def "publishes signature files for Maven publication"() {
+        given:
+        buildFile << """
+            apply plugin: 'maven-publish'
+            ${keyInfo.addAsPropertiesScript()}
+
+            task sourceJar(type: Jar) {
+                from sourceSets.main.allJava
+            }
+
+            publishing {
+                publications {
+                    mavenJava(MavenPublication) {
+                        from components.java
+                        artifactId '$artifactId'
+                        artifact sourceJar {
+                            classifier "sources"
+                        }
+                    }
+                }
+                repositories {
+                    maven {
+                        name "m2"
+                        url "file://\$buildDir/m2Repo/"
+                    }
+                }
+            }
+
+            signing {
+                ${signingConfiguration()}
+                sign publishing.publications.mavenJava
+            }
+        """
+
+        and:
+        enableGradleMetadata()
+
+        when:
+        succeeds "publishMavenJavaPublicationToM2Repository"
+
+        then:
+        ":publishMavenJavaPublicationToM2Repository" in nonSkippedTasks
+
+        and:
+        pom().assertExists()
+        pomSignature().assertExists()
+        m2RepoFile(jarFileName).assertExists()
+        m2RepoFile("${jarFileName}.asc").assertExists()
+        m2RepoFile("$artifactId-${version}-sources.jar").assertExists()
+        m2RepoFile("$artifactId-${version}-sources.jar.asc").assertExists()
+        m2RepoFile("$artifactId-${version}.module").assertExists()
+        m2RepoFile("$artifactId-${version}.module.asc").assertExists()
+    }
+
+    @IgnoreIf({GradleContextualExecuter.parallel})
```

Comment:
Why this ignore?

Reply:
Very good question... I started off with a test from a different class where all methods had that annotation. I'll remove it and check if it works.

Reply:
Remove if `@IgnoreIf` in 069edac8508.